skrf/calibration/deembedding.py

```diff
@@ -236,17 +236,16 @@ def deembed(self, ntwk):
 
         # check if the frequencies match with dummy frequencies
         if ntwk.frequency != self.open.frequency:
-            raise(ValueError('Network frequencies dont match dummy frequencies.'))
-
-        # TODO: attempt to interpolate if frequencies do not match
-
-        caled = ntwk.copy()
+            warnings.warn('Network frequencies dont match dummy frequencies, attempting overlap.', RuntimeWarning)
+            caled, op, sh = overlap_multi([ntwk, self.open, self.short])
+        else:
+            caled, op, sh = ntwk.copy(), self.open, self.short.copy()

```

Comment:
This code certainly works, but it might confuse ```self.short``` and ```deembeded_short```.
How about the following implementation?
```
caled, op, sh = ntwk.copy(), self.open, self.short

deembeded_short = sh.copy()
deembeded_short.y = sh.y - op.y
```

Reply:
Yep, thanks for pointing that out, I think this is a better way to do it!